organisation/2018-06-06_howToMakeDecisions.md

```diff
@@ -0,0 +1,32 @@
+# Structure proposal
+## Participants
+[Timo Mayer](https://github.com/timomayer)
+[Marc Wißler](https://github.com/SheepFromHeaven)
+
+## Why?
+We were talking about the need of a structured organisation because of the following needs:
+
+- Insecurity (Am I developing in the right direction?)
+- Decision making (Should this feature be worked on?)
+- Finance (Do we have budget for this task?)
+- Overview (What happens in the other repos?)
+
+## Result
+
+Our current proposal for a organisational setup is, that we will have three dedicated roles:
+
+1. The captain
+This person will manage the whole budget pool for all of biotope and will make sure, the overall project heads in the right direction.
+
+2. The officers
+These persons will be given a small cut of the budget to decide on how to separate this budget over the different projects.
+
+3. The navigators
+These persons are experienced in the distinct projects and can ship the progress in the right directions. These are the first go-tos for questions.
+
+
+Every project (boilerplate, build, generator, etc.) should have one officer and one navigator assigned to it.
+One navigator/officer can have multiple projects she cares for.
+
+
+This structure shall be kept until the projects work on their own.
```

Comment:
steering meeting: There is a monthly steering meeting with all officers and the captain to navigate on (time, budget, quality, next steps)